client-misk-dynamodb/src/test/kotlin/app/cash/backfila/client/misk/dynamodb/DynamoDbFilteringTest.kt

```diff
@@ -65,8 +65,10 @@ class DynamoDbFilteringTest {
     }
 
     companion object {
-      val TRACK_SORT_START = "TRACK_"
+      const val TRACK_SORT_START = "TRACK_"
     }
+
+    override fun fixedSegmentCount(config: BackfillConfig<NoParameters>): Int? = 16
```

Comment:
Makes the tests run much faster. The default of 2048 is a better starting point for realz though.